hera/ui/PlayerCard.tsx

```diff
@@ -289,6 +314,61 @@ export default memo(function PlayerCard({
                   );
                 })}
             </Stack>
+            {wide && (
+              <Stack className={offsetStyle} gap nowrap start>
+                {[
+                  [
+                    'fundsPerTurn',
+                    shouldShow ? calculateFunds(map, player) : '???',
+                    <Icon
+                      className={playerStatsAfterIconStyle}
+                      icon={Reload}
+                      key={0}
+                    />,
+                  ] as const,
+                  [
+                    'netUnits',
+                    shouldShow
+                      ? Math.max(
+                          player.stats.createdUnits - player.stats.lostUnits,
+                          0,
+                        )
+                      : '???',
+                    null,
+                  ] as const,
+                  [
+                    'netBuildings',
+                    shouldShow
+                      ? Math.max(
+                          player.stats.createdBuildings -
+                            player.stats.lostBuildings,
+                          0,
+                        )
+                      : '???',
+                    null,
+                  ] as const,
+                ].map(([key, value, after]) => {
+                  const icon =
+                    key === 'fundsPerTurn'
+                      ? Coin
+                      : key === 'netUnits'
+                        ? HumanHandsdown
+                        : Buildings;
```

Comment:
Let's make the icon the third argument instead of checking here. Also, mind calling it just `units` and `buildings` instead of `netUnits` etc.?